js/gracelib.js

```diff
@@ -3036,7 +3036,7 @@ function callmethod(obj, methname, argcv) {
         } else if (e.exctype === 'graceexception') {
             e.exitStack.unshift({
                 className: obj.className,
-                methname: methname,
+                methname: canonicalMethodName(methname),
```

Comment:
Doing this here is slow.  If the exception is caught, the exitStack frame will be discarded.  It's better to do it in GraceCallStackToString (on line 2987), when the error string is created from the exit stack frame object.